enhancements/update/upgrade-without-registry.md

```diff
@@ -0,0 +1,893 @@
+---
+title: upgrade-without-registry
+authors:
+- "@jhernand"
+reviewers:
+- "@avishayt"
+- "@danielerez"
+- "@mrunalp"
+- "@nmagnezi"
+- "@oourfali"
```

Comment:
Given the number of CVO changes proposed, we should have someone from the OTA team look at this, too. 

cc @LalatenduMohanty

Reply:
@jhernand can you add me to approver's list?

Reply:
Yes, will be in the next version of the PR.